api/http/dapps.js

```diff
@@ -41,24 +27,10 @@ function DappsHttpApi (dappsModule, app) {
 
 	router.map(dappsModule.internal, {
 		'get /categories': 'categories',
-		'get /installed': 'installed',
-		'get /installedIds': 'installedIds',
-		'get /ismasterpasswordenabled': 'isMasterPasswordEnabled',
-		'get /installing': 'installing',
-		'get /uninstalling': 'uninstalling',
-		'get /launched': 'launched',
-		'post /launch': 'launch',
-		'put /transaction': 'addTransactions',
-		'put /withdrawal': 'sendWithdrawal'
 	});
 
 	router.get('/', httpApi.middleware.sanitize('query', schema.list, dappsModule.internal.list));
-	router.put('/', httpApi.middleware.sanitize('body', schema.put, dappsModule.internal.put));
-	router.get('/get', httpApi.middleware.sanitize('query', schema.get, dappsModule.internal.get));
-	router.get('/search', httpApi.middleware.sanitize('query', schema.search, dappsModule.internal.search));
-	router.post('/install', httpApi.middleware.sanitize('body', schema.install, dappsModule.internal.install));
-	router.post('/uninstall', httpApi.middleware.sanitize('body', schema.uninstall, dappsModule.internal.uninstall));
-	router.post('/stop', httpApi.middleware.sanitize('body', schema.stop, dappsModule.internal.stop));
+	router.get('/:id', httpApi.middleware.sanitize('param', schema.get, dappsModule.internal.get));
```

Comment:
This is a premature change of API. Not covered in tests.

Removing: `router.get('/get', httpApi.middleware.sanitize('query', schema.get, dappsModule.internal.get));`